shop/src/pages/Order.js

```diff
@@ -160,7 +158,7 @@ const Order = () => {
       if (result) {
         setCart(result.cart)
         setError(false)
-        dispatch({ type: 'orderComplete' })
+        // dispatch({ type: 'orderComplete' })
```

Comment:
nit: clean up?

Reply:
Ah yes, I actually commented it while testing locally, adding it back